### DISCUSSION TITLE
Is it possible to write a little more content about the custom page?

### DISCUSSION THREAD

**Discussion by dongrentianyu** [web-infra-dev/rspress#516] [Q&A]
目前的想法是想复刻一个rspress-doc主题，但里面有些样式不一样。除了样式不一样，其他的基本上和repress-doc一样。

```
.rspress-doc {
  text-align: center;
  white-space: pre-line;
}
```

样式是需要保持居中，和硬换行。其他的没了。

但没看到如何具体地自定义页面。因为一旦设置成`pageType: custom`，基本上就是纯文本了。

如果我直接把上面的代码写在全局加载里，那么就不能写正常的博客文本了。因为博客文本也会变成居中的形式。

**Comment by dongrentianyu** [comment]
我把上面的样式改成`rspress-poem`。

```
.rspress-poem {
  text-align: center;
  white-space: pre-line;
}
```

然后通过在Front Matter里增加className，如下所示。但没有成功。这是为什么呢？

```
---
title: 几十年的一个夜晚
className: rspress-poem
---
```

**Reply by sanyuan0704** [reply]
https://rspress.dev/zh/guide/basic/custom-page.html#%E8%87%AA%E5%AE%9A%E4%B9%89%E6%A0%B7%E5%BC%8F 通过自定义全局样式就可以，把原来  .rspress-doc 的样式进行覆盖

**Reply by dongrentianyu** [reply]
我是想保留两种，一种是原生态的rspress-doc样式，一种是在原生态rspress-doc后，居中且硬换行的。如果我用全局样式覆盖掉了，正常写文章也会变得居中了。

**Reply by sanyuan0704** [reply]
我理解你的意思了，我可以在主题里面支持这种 frontmatter 写法
